File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/SafetyReportHarshEvent.scala

package org.openapitools.client.model


case class SafetyReportHarshEvent (
    /* Type of the harsh event */
    _harshEventType: Option[String],
    /* Timestamp that the harsh event occurred in Unix milliseconds since epoch */
    _timestampMs: Option[Integer],
    /* Vehicle associated with the harsh event */
    _vehicleId: Option[Integer]
)
object SafetyReportHarshEvent {
    def toStringBody(var_harshEventType: Object, var_timestampMs: Object, var_vehicleId: Object) =
        s"""
        | {
        | "harshEventType":$var_harshEventType,"timestampMs":$var_timestampMs,"vehicleId":$var_vehicleId
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/SafetyReportHarshEvent.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * List of harsh events
 * @param harshEventType Type of the harsh event
 * @param timestampMs Timestamp that the harsh event occurred in Unix milliseconds since epoch
 * @param vehicleId Vehicle associated with the harsh event
 */
data class SafetyReportHarshEvent (

        @JsonProperty("harshEventType") val harshEventType: String? = null,

        @JsonProperty("timestampMs") val timestampMs: Int? = null,

        @JsonProperty("vehicleId") val vehicleId: Int? = null
) {

}

